### DISCUSSION TITLE
Cropping to center after rotation returns unexpected area

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#2285] [Help]
I want to rotate some images that only contain relevant information in the center. The images are of size 101x101 and each image should be rotated by 1 to 360 degrees. Rotating with 
`convert in.png -rotate angle out.png`
works fine and the images are enlarged as expected to stay rectangular. For example an angle of 44 returns an: PNG image data, 145 x 145, 8-bit grayscale, non-interlaced
Since I only care about the center part and need images with uniform format I try to crop to 70x70 after, using:
`convert out.png -gravity center -crop 70x70+0+0 out_crop.png`
But the results are not as expected. The images are not cropped to center, but rather off to the bottom left. I feel like it has something to do with the rotation, since cropping unrotated pictures returns the expected area. 

Has anyone any idea what might be the problem?

**Comment by snibgo** [comment]
The problem is that "-rotate" creates metadata about the image, which is used by "-crop". Remove this with "+repage" after "-rotate XX".

**Reply by ghost** [reply]
Thanks man, works like a charm!

If anyone is interested: Before I tried to append +repage to the crop command, looks like this does not work and the metadata is deleted after the failed crop
